### ISSUE TITLE
FATAL Invalid or unexpected token - getInfo()

### ISSUE THREAD

**Issue by Mittelblut9** [fent/node-ytdl-core#1123]
Hello everyone.
I got this error when i run my code (see below).
But it seems like a module error? Some suggestions how to fix this?

<img width="804" alt="Screenshot 2022-08-25 at 12 24 54" src="https://user-images.githubusercontent.com/67608550/186641631-6a70bd16-2200-4073-8c94-877d61604859.png">

```js
const video_link = "https://www.youtube.com/watch?v=BmjBU0IIR0k"

let info = await ytdl.getInfo(video_link);
```

**Comment by phaze-the-dumb** [comment]
Its throwing pretty much the same error for me even when I don't use `ytdl.getInfo()` when i'm just using `ytdl()` to download videos too

**Comment by 11208** [comment]
Can you follow the instruction in this issue to see if it fix your problem? [https://github.com/fent/node-ytdl-core/issues/1108](url)

**Comment by Mittelblut9** [comment]
@11208 Downgraded to 4.10.0. Now it works thanks! I hope it get fixed soon so i don't have to stay on a deprecated version :)

**Comment by TaylorDuck72** [comment]
Having same issue but not using `getInfo()`, it was working few hours ago.

**Comment by aandrew-me** [comment]
Rolling back to 4.10.0 works but the download speed is much slower.

**Comment by TimeForANinja** [comment]
marking this as a dup of #1108 or #1110 or #1113 
should also be fixed with #1126
